test/blueprint.test.ts

```diff
@@ -1,10 +1,16 @@
 import test from 'ava'
 
-import { createBlueprint } from '@seamapi/blueprint'
+import { createBlueprint, type TypesModule } from '@seamapi/blueprint'
+
+import type { OpenAPI } from 'lib/openapi.js'
 
 import * as types from './fixtures/types/index.js'
 
+const typesModule: TypesModule = {
+  openapi: types.openapi as unknown as OpenAPI,
+}
```

Comment:
Instead of type casting here, we should use `@ts-expect-error` since we need to eventually update the test fixture to match the type.